src/node/miner.cpp

```diff
@@ -158,10 +158,13 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock()
     CMutableTransaction coinbaseTx;
     coinbaseTx.vin.resize(1);
     coinbaseTx.vin[0].prevout.SetNull();
+    coinbaseTx.vin[0].nSequence = CTxIn::MAX_SEQUENCE_NONFINAL;
```

Comment:
2c10f93d0bd920f20e91f1be7c82e66af2a52919 nit: can you move this down next to `nLockTime`? This line is what makes the `nLockTime` bit work, so makes sense to have them together. Also, in https://github.com/Sjors/bitcoin/pull/49 I put this behavior behind a startup flag.

Reply:
I added the comment as you requested above (https://github.com/bitcoin/bitcoin/pull/32155#discussion_r2018964740), hopefully this makes it clearer.

I don't think there should be a startup flag since i don't see a reason for disabling it: this should not affect miners in any way.

Reply:
It affects any miner that uses the Mining interface, because `createNewBlock` returns a `BlockTemplate` which includes a `getBlock()` method which returns a `CBlock` that contains the coinbase transaction.